vpr/src/route/check_rr_graph.cpp

```diff
@@ -124,16 +124,16 @@ void check_rr_graph(const t_graph_type graph_type,
             t_rr_type to_rr_type = device_ctx.rr_nodes[to_node].type();
 
             /* Only expect the following cases to have multiple edges
-             * - chan <-> chan connections 
-             * - IPIN <-> chan connections (unique rr_node for IPIN nodes on multiple sides)
-             * - OPIN <-> chan connections (unique rr_node for OPIN nodes on multiple sides)
+             * - CHAN <-> CHAN connections
+             * - CHAN  -> IPIN connections (unique rr_node for IPIN nodes on multiple sides)
+             * - OPIN  -> CHAN connections (unique rr_node for OPIN nodes on multiple sides)
```

Comment:
My suggestion for clarity is to break this into many lines, e.g.:

```
bool is_chan_to_chan = ...;
bool is_chan_to_ipin = ...;
bool is_opin_to_chan = ...;
```

That will make the intention of each boolean statement clear, and easier to follow.  Then the final if statement will be very simple too!

Reply:
Sure, makes total sense. Done